src/com/google/javascript/jscomp/ConvertChunksToESModules.java

```diff
@@ -225,7 +233,22 @@ private void addImportStatements() {
   private class FindCrossChunkReferences extends AbstractPostOrderCallback {
     @Override
     public void visit(NodeTraversal t, Node n, Node parent) {
-      if (n.isName()) {
+      if (n.isScript()) {
+        JSModule chunk = t.getModule();
+        List<JSModule> chunkDependencies = chunk.getDependencies();
+
+        // Ensure every chunk dependency is explicitly listed with an import
+        // Dependent chunks may have side effects even if there isn't an explicit name reference
+        if (chunkDependencies.size() > 0) {
```

Comment:
The "Error Prone" tool complains that this would be better as `!chunkDependencies.isEmpty()`.
Any reason not to change to that?

Reply:
That would be equivalent and better